controllers/argocd/util.go

```diff
@@ -36,7 +36,6 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	v1 "k8s.io/api/rbac/v1"
```

Comment:
Do we want to keep it `v1` or update it to `rbacv1` as imports in other files?

Reply:
`util.go` will eventually go away so we can leave it as is